policyengine_us/parameters/gov/states/household/state_eitcs.yaml

```
@@ -16,9 +16,9 @@ values:
     - md_eitc  # Maryland
     - me_eitc  # Maine
     - mi_eitc  # Michigan
-    - mn_wfc  # Minnesota (called “Working Family Credit”) repealed in 2023
+    - mn_wfc  # Minnesota (called “Working Family Credit") repealed in 2023
```

While at it please replace the other direction of quotes
```suggestion
    - mn_wfc  # Minnesota (called "Working Family Credit") repealed in 2023
```